File: java/oop/HumanApp/src/com/techlabs/human/test/TestHuman.java
package com.techlabs.human.test;

import com.techlabs.human.GenderType;
import com.techlabs.human.Human;

public class TestHuman{
	public static void main(String[] args){
		Human human1=new Human("Chandan",22,5.3f,55.0f,GenderType.MALE);
		Human human2=new Human("foo",GenderType.FEMALE,19);
		
		//for human 1
		human1.eat();
		human1.workout();
		human1.workout();
		human1.eat();
		human1.eat();
		
		//for human 2
		human2.eat();
		human2.eat();
		human2.workout();		
		
		printDetails(human1);
		printDetails(human2);
	}
	
	private static void printDetails(Human human){
		System.out.println("Name = "+human.getName()
				+"\nAge = "+human.getAge()
				+"\nGender = "+human.getGender()
				+"\nHeight = "+human.getHeight()
				+"\nWeight = "+human.getWeight()
				+"\nYour Health is "+human.healthRecommendation());
		System.out.println("-----------------------------");
	}
}

File: dotNet/CSharp/HumanApp/HumanApp/Test/TestHuman.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace HumanApp.Test
{
    class TestHuman
    {
        static void Main(string[] args)
        {
            Human human = new Human("Chandan", GenderType.MALE, 22);

            human.eat();
            human.eat();
            human.eat();
            human.eat();
            human.eat();
            human.workout();
            human.workout();
 
            Console.WriteLine(human);
        }
    }
}
